libs/ui-lib/lib/ocm/components/clusterConfiguration/staticIp/commonValidationSchemas.tsx

```diff
@@ -112,25 +139,48 @@ export const isReservedIpv6Address = (ipv6Address: Address6) => {
   return ipv6Address.isLoopback() || ipv6Address.isMulticast();
 };
 
-export const isNotReservedHostIPAddress = (
-  protocolVersion: ProtocolVersion,
-  isDnsAddress = false,
-) => {
-  const textForAddress = isDnsAddress ? 'a DNS' : 'an interface';
+export const isNotReservedHostIPAddress = (protocolVersion?: ProtocolVersion) => {
+  return Yup.string().test(
+    'is-not-reserved-ip-address',
+    ({ value }) => {
+      const addresses = (value as string).split(',');
+      if (addresses.length === 1) {
+        return `Provided IP address is not a correct address for an interface.`;
+      }
```

Comment:
What are you testing here? The msg doesn't make...
e.g.: `'192.168.1.1'`=> `addresses.length === 1` => `Provided IP address is not a correct address for an interface.` Why not?

Reply:
because the IP can be reserved

`'255.255.255.255'=> addresses.length === 1 => Provided IP address is not a correct address for an interface.`